bridge/test/contract-mocks/snapshots/SnapshotsMock.sol

```diff
@@ -203,4 +203,25 @@ contract SnapshotsMock is Initializable, ImmutableValidatorPool, ISnapshots {
         desperationFactor;
         return true;
     }
+
+    function checkBClaimsSignature(bytes calldata groupSignature_, bytes calldata bClaims_)
+        public
+        pure
+        returns (bool)
+    {
+        groupSignature_;
+        bClaims_;
```

Comment:
If this is because of a linter warning for unused variables there might be a different workaround. Possibly removing the name from the signature, or replacing with a `_`:

```solidity
function checkBClaimsSignature(bytes calldata, bytes calldata)
```

Reply:
Yep, this is to silence the linter. Unfortunately, we cannot change the signature (both name and variables) because this Mock needs to conform to the `ISnapshots` solidity interface.